app/views/links/_left_sidebar.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="col-lg-1 col-md-1 ">
```

Comment:
`_left_sidebar.html.erb` is html for sidebar which would be common to all pages. Hence it should reside in `shared` directory